docs/src/main/tut/chisel3/resources.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: docs
+title:  "Resources and References"
+section: "chisel3"
+---
+
+The *best resource* to learn about Chisel is the [**online Chisel Bootcamp**](https://mybinder.org/v2/gh/freechipsproject/chisel-bootcamp/master). This runs in your browser and assumes no prior Scala knowledge.
```

Comment:
Shouldn't we also link to the repo too in case users want to download it? Once the binder instance expires there's no way to get the data in it back

Reply:
This is copying how the current chisel readme presents it. That's a bit terse, so I think it may make sense to link both. I'm more concerned about barrier to access where windows users just want to learn something, but they go down a rabbit hole of a complicated installation.

Reply:
I think it makes sense to make the binder link the most prominent here, though we could (should?) add a link to the repo at the top of the binder notebook or something - "your work on Binder is not saved, you can run the notebook locally or view the source at ..."

I'd prefer making the easy path as easy and obvious as possible, but still make it possible to find sources / harder paths in a way that's not confusing (eg, footnotes, small text, etc).

Reply:
I added some clarifying text that users may also run this locally.